lib/ansible/modules/monitoring/zabbix/zabbix_proxy.py

```diff
@@ -114,8 +153,6 @@
     status: active
     state: present
     interface:
-        type: 0
```

Comment:
Is there a reason this is removed (and not documented above)?

Reply:
Yes, it is unused. Proxy interface is different from Host interface. Parameters `type` and `main` were never supported by Zabbix API. [Doc for reference](https://www.zabbix.com/documentation/4.0/manual/api/reference/proxy/object#proxy_interface)